py-polars/polars/internals/expr/datetime.py

```diff
@@ -383,6 +412,35 @@ def iso_year(self) -> pli.Expr:
         -------
         ISO Year as Int32
 
+        Examples
+        --------
+        >>> from datetime import timedelta, datetime
+        >>> start = datetime(2001, 1, 1)
+        >>> stop = datetime(2006, 1, 1)
+        >>> df = pl.DataFrame({"date": pl.date_range(start, stop, timedelta(days=180))})
```

Comment:
You seem to be a big fan of `date_range` 😄 I think the examples are fine, but would probably be clearer if you just hardcoded three datetimes in a dataframe. You already specified a start and stop, you're 2/3rds of the way there 😄

Reply:
Yeah I could do that, but most of the other examples for expressions use date_range to make a couple examples, so I used this so it's in line with the other examples.

Reply:
But I agree I can trim down the amount of rows in this example, the amount now is a bit overkill